linalg/ginkgo.cpp

```diff
@@ -21,15 +21,24 @@
 #include <iomanip>
 #include <algorithm>
 #include <cmath>
+#include <cstring>
 
 namespace mfem
 {
 
 namespace Ginkgo
 {
 
+// Create a GinkgoExecutor of type exec_type.
 GinkgoExecutor::GinkgoExecutor(ExecType exec_type)
 {
+   gko::version_info gko_version = gko::version_info::get();
+   bool gko_omp_status = true;
```

Comment:
`bool gko_omp_status = (strcmp(gko_version.omp_version.tag, "not compiled") != 0)`

Reply:
This was fixed in 1ae3aad